.github/workflows/push.yml

```diff
@@ -14,3 +14,5 @@ jobs:
       run-check-licenses: true
       run-format-check-c-cpp: true
       run-format-check-md: true
+      use-sccache: ${{github.actor != 'dependabot[bot]'}}
+      sccache-write: true
```

Comment:
Once stuff merges then we write the new objects.

Reply:
I think that makes a lot of sense anyway. It's unlikely that prs will depend on other prs so we won't get a speedup anyway